modules/publinkIdSystem.js

```diff
@@ -137,9 +145,7 @@ export const publinkIdSubmodule = {
     if (localValue) {
       return {id: localValue};
     }
-    if (!storedId) {
-      return {callback: makeCallback(config, consentData)};
-    }
+    return {callback: makeCallback(config, consentData, storedId)};
```

Comment:
Maybe we can skip callbacks if there is no api key to begin with.  This way if the standalone script was the one that generated the id, then this module doesn't need to bother with refresh.

Reply:
if we skip the callback those potential requests dissappear going unnoticed.  If I make the callout we have a record of it in the logs missing the api key

Reply:
That's what we want, 'cause most prebid users won't have api key configured.

Reply:
If you are worried about the initial generate call, then we can improve the check by skipping refresh without apikey.  In other words, if there is a storedId already but no api key, then it can be safely skipped.

Reply:
It's not that.  When requests drop or are not coming in it's better for us if client side code isn't suspected as a point of failure for not sending the request through, Just let it pass through and leave the decision making to the backend.